reviews/models/mixins.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from transitions import Machine
+
+from django.db import models
+
+from reviews import workflow
+
+
+class ReviewProviderMixin(models.Model):
```

Comment:
We could do that with a new `ReviewProviderSettings` model. That seems cleaner than the mixin, especially as the amount of reviews-related metadata grows. Also would be nicer if/when we have multiple types of providers.